packages/scene-composer/src/store/slices/ViewOptionStateSlice.spec.ts

```diff
@@ -1,7 +1,12 @@
 import { ITagSettings, KnownComponentType } from '../../interfaces';
 import { Component } from '../../models/SceneModels';
 
-import { createViewOptionStateSlice } from './ViewOptionStateSlice';
+import { createViewOptionStateSlice as _createViewOptionStateSlice } from './ViewOptionStateSlice';
+
+const createViewOptionStateSlice = (set) => {
+  const { noHistoryStates } = _createViewOptionStateSlice(set);
+  return noHistoryStates;
```

Comment:
This just to limit the return scope down to the noHistoryState parameter instead of the full list that _create gives? What's the reason for doing that?